tests/test_vision_train.py

```diff
@@ -87,3 +95,8 @@ def test_preds(learn):
 
 def test_lrfind(learn):
     learn.lr_find(start_lr=1e-5,end_lr=1e-3, num_it=15)
+
+@pytest.mark.parametrize('arch', [models.resnet18, models.squeezenet1_1])
+def test_models_meta(mnist_tiny, arch, image):
```

Comment:
By default this uses a pretrained model, which will try to download the whole weight file from the internet. We want our tests to run quickly and without internet, so you shouldn't use pretrained models in your tests unless you mark them as 'slow'.

Reply:
Ok, understood. The only reason to use the `torchvision` models was to verify that `create_cnn` still works. Would you propose to mark them 'slow' or replace all together with something computationally efficient?

Reply:
Done!